tests/Feature/LoginPageTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+
+class LoginPageTest extends TestCase
+{
+
+    use DatabaseMigrations;
+
+    private $user = null;
+    private $centre = null;
+
+    public function seedForTests()
```

Comment:
If you put this in a setup function it will run for every test.

Reply:
For some reason I thought setUp() ran before the DatabaseMigrations trait had done it's magic. Maybe the interwebs lied to me. changed!